.github/workflows/ci-lint.yml

```
@@ -0,0 +1,29 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: ci-lint
+
+# Trigger the workflow when:
+on:
+  # A push occurs to one of the matched branches.
+  push:
+    branches: [master]
+  # Or when a pull request event occurs for a pull request against one of the
+  # matched branches.
+  pull_request:
+    branches: [master]
+
+jobs:
+  lint:
+    # NOTE: This name appears in GitHub's Checks API.
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Node.js 18
+        uses: actions/setup-node@v3.5.0
```

Let's try not locking it to a concrete version so we don't need to update it all the time:

```suggestion
        uses: actions/setup-node@v3
```

Also, version `3.5.0` is causing a lot of warning because of https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/.